### PR TITLE
fix: asset list page

### PR DESCRIPTION
**Pull request by Space-zee**

### CODE REVIEW

File: frontend/app/features/assets/views/ProgressBar/ProgressBarTextInside/index.tsx [hunk 1]
```diff
@@ -1,17 +1,25 @@
-import { FC } from 'react';
+import React, {FC} from 'react';
```
Comment (dmitriy-green) [inline frontend/app/features/assets/views/ProgressBar/ProgressBarTextInside/index.tsx:1]: why importing 'React'?

Comment (Space-zee) [reply frontend/app/features/assets/views/ProgressBar/ProgressBarTextInside/index.tsx]: accidentally

Comment (Space-zee) [reply frontend/app/features/assets/views/ProgressBar/ProgressBarTextInside/index.tsx]: removed

File: frontend/app/features/assets/views/AssetList/index.tsx [hunk 2]
```diff
@@ -3,53 +3,181 @@ import AssetListItem from "./AssetListItem";
...
+  activeFilter: AssetStatuses;
 };
 
-const AssetList = ({assetsCombined}: Props) => {
+const AssetList = ({
+                     assetsCombinedActiveUpcoming,
+                     assetsCombinedSoldOut,
+                     assetsCombinedOther,
+                     assetsCombined,
+                     activeFilter
```
Comment (ruslanpalagin) [inline frontend/app/features/assets/views/AssetList/index.tsx:22]: Do NOT pass activeFilter.
You MAY pass title as a text - just to render. 
Move activeFilter into model. Use to generate

**Review by ruslanpalagin — DISMISSED** [review]

**Pull request merged**